src/main/java/greencity/service/impl/EmailServiceImpl.java

```diff
@@ -21,45 +24,36 @@
  * {@inheritDoc}
  */
 @Service
+@RequiredArgsConstructor
 @Slf4j
 public class EmailServiceImpl implements EmailService {
     private static final String EMAIL_CONTENT_TYPE = "text/html; charset=utf-8";
-    //subjects
+    // subjects
     private static final String GC_CONTRIBUTORS = "GreenCity contributors";
+    private static final String NEW_PLACES = "New places";
     private static final String VERIFY_EMAIL = "Verify your email address";
     private static final String CONFIRM_RESTORING_PASS = "Confirm restoring password";
-    //params
+    // params
     private static final String CLIENT_LINK = "clientLink";
     private static final String USER_NAME = "userFirstName";
     private static final String PLACE_NAME = "placeName";
     private static final String STATUS = "status";
     private static final String VERIFY_ADDRESS = "verifyAddress";
     private static final String RESTORE_PASS = "restorePassword";
-    //template names
+    private static final String RESULT = "result";
+    private static final String REPORT_TYPE = "reportType";
+    // templates
     private static final String CHANGE_PLACE_STATUS_EMAIL_PAGE = "change-place-status-email-page";
     private static final String VERIFY_EMAIL_PAGE = "verify-email-page";
     private static final String RESTORE_EMAIL_PAGE = "restore-email-page";
+    private static final String NEW_PLACES_REPORT_EMAIL_PAGE = "new-places-report-email-page";
```

Comment:
Probably move constants to separate class?

Reply:
Okay, looks nice, I will fix it